docs/reference/troubleshoot.md

```diff
@@ -7,12 +7,22 @@ indent: true
 
 # Troubleshooting
 
+* [Requested Resource Not Found]
 * [Resource Status and Conditions]
 * [Resource Events]
 * [Crossplane Logs]
 * [Pausing Crossplane]
 * [Deleting a Resource Hangs]
 
+## Requested Resource Not Found
+
+If you use kubectl Crossplane plugin to install a `Provider` or `Configuration` -
+e.g. `kubectl crossplane install provider crossplane/provider-aws:master` and
+get `the server could not find the requested resource` error, more often than
+not, that is an indicator that the kubectl Corssplane you're using is outdated.
+In other words some Crossplane API has been graduated from alpha to beta or stable
+and the old plugin is not aware of this change.
+
```

Comment:
Maybe link back to the CLI install instructions here to give folks a way to fix the problem

Reply:
I added the link, but the `v1.0` is hardcoded in the URL which is not very convenient. I don't think there's a redirect from `https://crossplane.io/docs/FOO => https://crossplane.io/docs/<version>/FOO`, is there?

Reply:
I believe you can use a relative path link :)

```
../getting-started/install-configure.md#install-crossplane-cli
```

Reply:
Ha, good to know!